tests/integration/nodejs/promise-message/index.js

```diff
@@ -12,7 +12,7 @@ function sleep(ms) {
     return new Promise((resolve) => setTimeout(resolve, ms));
 }
 
-sleep(2000).then(() => {
+sleep(60000).then(() => {
```

Comment:
We want to leak this promise, the test will complete faster than the time we wait here because we call `exit(0)` below